x-pack/plugins/ml/public/application/datavisualizer/index_based/components/field_count_panel/field_count_panel.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiNotificationBadge, EuiSwitch, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React, { FC } from 'react';
+
+interface Props {
+  metricsStats?: {
+    visibleMetricFields: number;
+    totalMetricFields: number;
+  };
+  fieldsCountStats?: {
+    visibleFieldsCount: number;
+    totalFieldsCount: number;
+  };
+  showEmptyFields: boolean;
+  toggleShowEmptyFields: () => void;
+}
+export const FieldCountPanel: FC<Props> = ({
+  metricsStats,
+  fieldsCountStats,
+  showEmptyFields,
+  toggleShowEmptyFields,
+}) => {
+  return (
+    <EuiFlexGroup
+      alignItems="center"
+      gutterSize="xs"
+      style={{ marginLeft: 5 }}
+      data-test-subj="mlDataVisualizerFieldCountPanel"
+    >
+      {metricsStats && (
+        <EuiFlexGroup
+          gutterSize="s"
+          alignItems="center"
+          style={{ maxWidth: 200 }}
+          data-test-subj="mlDataVisualizerMetricFieldsSummary"
+        >
+          <EuiFlexItem grow={false}>
+            <EuiText>
+              <h5>
+                <FormattedMessage
+                  id="xpack.ml.dataVisualizer.searchPanel.metricsLabel"
+                  defaultMessage="Metrics"
```

Comment:
I know we used this label in the old page, but I wonder if something like `number fields` would make more sense, corresponding to the text in the field type dropdown?

Reply:
Updated here https://github.com/elastic/kibana/pull/85726/commits/bd93f5afb82a40117f16b37ec2337b6907715224
![Uploading Screen Shot 2020-12-14 at 10.27.20.png…]()